README.md

```diff
@@ -6,21 +6,44 @@ such as `rustc`, `cargo`, `rustup`, etc., as well as additional plugins and targ
 * `cargo-chef` - a tool for building and caching dependencies for a rust project
 * `cargo-sbom` - a tool for generating SBoMs for rust-based projects
 * targets to build for linux amd64, arm64 and riscv64 on linux for both musl and glibc
+* `mold` - a very fast linker for cross-compilation targets
+* `clang` - to invoke `mold` for cross-compilation targets
 
 ## Usage
 
 Use this image as a base `FROM` when building in your EVE Dockerfile. For example:
 
 ```Dockerfile
-FROM lfedge/eve-rust:1.80.1 AS rust
+# we use host tools to avoid emulation and slow builds
```

Comment:
While this all is correct, it also is confusing, while the original example is straightforward to understand.  Can we leave the original example, and then add a section for cross-compilation?

Reply:
> While this all is correct, it also is confusing, while the original example is straightforward to understand. Can we leave the original example, and then add a section for cross-compilation?

ok, I'll create a separate example